ocplib-json-typed-browser.opam

```diff
@@ -1,18 +1,21 @@
 opam-version: "2.0"
 name: "ocplib-json-typed-browser"
-synopsis: "Json_repr interface over JavaScript's objects"
-version: "0.7.1"
+synopsis: "Libraries for reliable manipulation JSON objects (browser support)"
+version: "dev"
 maintainer: "Benjamin Canou <benjamin@ocamlpro.com>"
 authors: "Benjamin Canou <benjamin@ocamlpro.com>"
 homepage: "https://github.com/ocamlpro/ocplib-json-typed"
 bug-reports: "https://github.com/ocamlpro/ocplib-json-typed/issues"
 license: "LGPLv3 w/ linking exception"
 dev-repo: "git+https://github.com/ocamlpro/ocplib-json-typed.git"
 
-build: [ "dune" "build" "-j" jobs "-p" name "@install" "@runtest" {with-test} ]
+build: [
+  ["dune" "build" "-j" jobs "-p" name "@install"]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
 depends: [
-  "ocaml" {>= "4.03.0"}
-  "dune" {build & >= "1.0.1"}
-  "ocplib-json-typed" {= "0.7.1" }
+  "ocaml" {>= "4.3.0"}
```

Comment:
I'm not sure on that change. I think it might be ok and 4.03.0 >= 4.3.0 is true, but I would prefer to keep that line.

Reply:
done